File: aat-android/src/main/java/ch/bailu/aat/views/msg/overlay/TipMsgView.java
package ch.bailu.aat.views.msg.overlay;

import android.content.Context;

import ch.bailu.aat.util.ui.tooltip.ToolTip;
import ch.bailu.aat.views.msg.AbsMsgView;

public class TipMsgView extends AbsMsgView {
    private final static int DISPLAY_FOR_MILLIS = 6000;

    public TipMsgView(Context context) {
        super(context, DISPLAY_FOR_MILLIS);
        ToolTip.themeify(this);
    }

    @Override
    public void attach() {}

    @Override
    public void detach() {}
}

File: aat-android/src/main/kotlin/ch/bailu/aat/views/msg/overlay/TipMsgView.kt
package ch.bailu.aat.views.msg.overlay

import android.content.Context
import ch.bailu.aat.util.ui.tooltip.ToolTip
import ch.bailu.aat.views.msg.AbsMsgView

class TipMsgView(context: Context) : AbsMsgView(context) {
    init {
        ToolTip.themeify(this)
    }

    override fun attach() {}
    override fun detach() {}
}
